cds/modules/theme/static/templates/cds/marc21/default.html

```diff
@@ -0,0 +1,29 @@
+<li>
+  <h4><a href="/record/{{ record._id }}">{{ record._source.title_statement.title }}</a></h4>
```

Comment:
This commit was just to have the docker container running.
We will start checking the code later but thanks for pointing this out :smiley: